app/views/publish/courses/subjects/_primary_form.html.erb

```diff
@@ -10,7 +10,7 @@
     </legend>
     <%= render "publish/shared/error_messages", error_keys: [:subjects] %>
     <% primary_form_options.each do |opt| %>
-      <%= form.govuk_radio_button :master_subject_id, opt.code, label: { text: opt.name } %>
+      <%= form.govuk_radio_button :master_subject_id, opt.id, label: { text: opt.name } %>
```

Comment:
Can we keep the implementation same/similar to the secondary subjects version 


Using
```
course.selectable_subjects
```

Reply:
Even better